src/keysym.h

```diff
@@ -57,13 +57,24 @@
  */
 /** Minimum keysym value */
 #define XKB_KEYSYM_MIN            0x00000000
+/** Minimum keysym value assigned */
+ #define XKB_KEYSYM_MIN_ASSIGNED  0x00000000
```

Comment:
I have no objection, but what is the added value?

Reply:
readability, primarily ("oh, this zero is a keysym!") but also *some* typesafety because `xkb_keysym_t` is a `uint32_t` whereas `0x0` is a plain boring `int`.